### ISSUE TITLE
Invalid Ajax response unable to solve this issue with previous suggestions

### ISSUE THREAD

**Issue by E-Mulas** [getgrav/grav-plugin-admin#2118]
I'm getting the message Invalid Ajax response after upgrading to the latest stable version of grav. I've tried to change Remote Fetch Method and Remote Verify Peer (SSL) as suggested in other posts but nothing happens, the issue is still there and I'm not able to install/unistall any plugin.

**Comment by rhukster** [comment]
We've had to revert Grav 1.7.11/Admin 1.10.11 as there were several issues.  Please revert back to 1.7.10/1.10.11 if possible.  CLI might be simplest:

```
bin/gpm direct-install https://getgrav.org/download/plugins/admin/1.10.10
bin/gpm direct-install https://getgrav.org/download/core/grav-update/1.7.10
```

**Comment by E-Mulas** [comment]
thank you, I've reverted back to 1.7.10, and I'm not getting the Invalid Ajax response anymore.  Today I've seen there is a new version (Grav v1.7.12). Should I upgrade from 1.7.10 or I'm gonna face the same issue? I mean is the 1.7.12 already considered a stable version, or better to wait?

**Comment by rhukster** [comment]
Those issues should be fixed in 1.7.12
